pkg/apis/v1alpha1/register.go

```diff
@@ -51,23 +51,41 @@ var (
 	AMIFamilyBottlerocket = "Bottlerocket"
 	AMIFamilyAL2          = "AL2"
 	AMIFamilyUbuntu       = "Ubuntu"
+	AMIFamilyWindows2019  = "Windows2019"
+	AMIFamilyWindows2022  = "Windows2022"
 	AMIFamilyCustom       = "Custom"
 	SupportedAMIFamilies  = []string{
 		AMIFamilyBottlerocket,
 		AMIFamilyAL2,
 		AMIFamilyUbuntu,
+		AMIFamilyWindows2019,
+		AMIFamilyWindows2022,
 		AMIFamilyCustom,
 	}
 	SupportedContainerRuntimesByAMIFamily = map[string]sets.String{
 		AMIFamilyBottlerocket: sets.NewString("containerd"),
 		AMIFamilyAL2:          sets.NewString("dockerd", "containerd"),
 		AMIFamilyUbuntu:       sets.NewString("dockerd", "containerd"),
+		AMIFamilyWindows2019:  sets.NewString("containerd"),
+		AMIFamilyWindows2022:  sets.NewString("containerd"),
+	}
+
+	Windows2019              = "2019"
```

Comment:
Do we need these values now that we are separating out the AMi families and we aren't supporting the variant or version labels?

Reply:
Used as const to create SSM query

Reply:
optional: If they're only being referenced in a single place, I'd prefer to not have them as constants in this file, but I'd also take the current state as-is